apps/vibe-coding-platform/app/api/chat/agent.ts

```diff
@@ -0,0 +1,64 @@
+import { convertToModelMessages, type UIMessageChunk } from 'ai'
+import { type ChatUIMessage } from '@/components/chat/types'
+import { getModelOptions } from '@/ai/gateway'
+import { tools } from '@/ai/tools'
+import { DurableAgent } from '@workflow/ai/agent'
+import { getWritable } from 'workflow'
+
+export async function codingWorkflow({
```

Comment:
happy to change the name. Looks like this rn in o11y

<img width="720" height="605" alt="Image" src="https://github.com/user-attachments/assets/d52f81f7-59a3-443e-b123-91bd450b668d" />